factory/post.js

```diff
@@ -26,7 +25,10 @@ function processContent(data) {
     var item = entries[i],
         postTitle = item.title.$t,
         content   = item.content.$t,
-        url       = item.link[4].href,
+        // Do not assume the link will always remain item 4 in the array
+        url       = _.filter(item.link, function (link) {
+          return link.rel === 'alternate' && link.type === 'text/html';
+        })[0].href,
```

Comment:
Lol, Google can decide to do anything, it's their product. Anyways, let's hope that doesn't happen. Nice one Chinenye!

Thanks for the PR, I'll merge it right now! 💯